src/Interface.js

```diff
@@ -0,0 +1,73 @@
+const root = document.documentElement;
+        
+const gridBlocks = document.getElementById("gridPage");
+let holding = false;
+
+// 2nd test object
+const ob = document.createElement("div");
+ob.className = "gridSpanTest";
+
+let row, column;
+let pos;
+const gridBlockLength = 2.34;
+
+//  Adding grid background
+for (let i = 0; i < 300; i++){
+    const gridSpace = document.createElement("div");
+    gridSpace.className = "gridBlock";
+    row = Math.floor(i/20)+1;
+    column = (i%20)+1;
+    pos = row.toString().concat(",", column.toString());
+    gridSpace.style.gridRowStart = row;
+    gridSpace.style.gridColumnStart = column;
+    gridSpace.id = pos;
+    gridSpace.addEventListener("click", place.bind(this, row, column));
+    gridBlocks.appendChild(gridSpace);
+}
+
+// creating test object
+const divSpan = document.createElement("div");
+const img = document.createElement("img");
+img.src = "images/swordVertical.jpg";
+img.className = "objectIcon";
+divSpan.className = "gridSpanTest";
+divSpan.appendChild(img);
+divSpan.style.width = ((gridBlockLength*2)-0.5).toString().concat("vw");
+divSpan.style.height = ((gridBlockLength*5)-0.5).toString().concat("vw");
+gridBlocks.appendChild(divSpan);
+
+// get mouse position inside the grid
+gridBlocks.addEventListener("mousemove", e => {
+    divSpan.style.setProperty('--mousex', e.clientX + "px");
+    divSpan.style.setProperty('--mousey', e.clientY + "px");
+});
+
+divSpan.addEventListener("click", handle);
+
+function handle() {
```

Comment:
A good name for a method usually includes both a verb (the action) and a noun (the target of the action). A name like 'handle' is fine if you only ever handle one thing, but will quickly need to be renamed when you need to handle two things. Even if you never handle more than one thing, when you come back to this in a year's time, you'll need to look through the code above to find out what this actually handles. If the method was named `handleGridBlockClick`, you'd know immediately what it does. 

Same for `place` below.